pkg/api/applications/v2/activity.go

```diff
@@ -107,6 +107,13 @@ type RefreshActivity struct {
 	ActivityFailure
 }
 
+type ActivityPatchRequest struct {
+	Title string   `json:"title"`
+	Tags  []string `json:"tags"`
```

Comment:
The client should never be able to change the tags (really, tag) on an activity: that should be determined by the server.

Reply:
That makes sense. I modeled it this way based on what the patch endpoint accepts, but based on your comment I would think we need an addtional level of authorization to only allow certain fields to be updated by the client. Will change this code to remove the tags bit, and can update app-svc accordingly.

Reply:
Updated - 705000cef444ffc108f984df0bad95ce9dda0c0e